tools/cert-sidecar/config/default.conf

```diff
@@ -0,0 +1,41 @@
+
+#URL of the x509 identity mgmt and Resources
+x509.url:string=http://x509-identity-mgmt:3000
+x509.path.sign:string=/internal/api/v1/throw-away
+x509.path.crl:string=/internal/api/v1/throw-away/ca/crl
+x509.path.ca:string=/internal/api/v1/throw-away/ca
+x509.retries:integer=9
```

Comment:
I calculated this time and thought it could be a good time, at least 8 min, but it can vary since the delay between attempts is random from what I could understand.

https://www.npmjs.com/package/axios-retry#options

https://www.wolframalpha.com/input/?i=Sum%5B1000*2%5Ek%2C+%7Bk%2C+0%2C+n-1%7D%5D+where+n%3D9

https://developers.google.com/analytics/devguides/reporting/core/v3/errors#backoff